local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,20 @@
+common:
+  namespace: default
+  defaultHostDir: /mnt/disks
+  defaultMountDir: /local-disks
+configmap:
+  configMapName: "test"
```

Comment:
I would use "local-provisioner-config" as the default name.

Reply:
done